packages/thirdweb/src/exports/react.ts

```diff
@@ -54,6 +54,8 @@ export {
   useIsAutoConnecting,
 } from "../react/core/hooks/wallets/wallet-hooks.js";
 
+export { type ConnectManagerOptions } from "../wallets/manager/index.js";
```

Comment:
```suggestion
export type { ConnectManagerOptions } from "../wallets/manager/index.js";
```

this way when importing only the type no ~200 extra bytes for the JS bundle (that ends up unused) are imported